src/core/distributions/phylogenetics/tree/birthdeath/BirthDeathSamplingTreatmentProcess.h

```diff
@@ -53,7 +53,8 @@ namespace RevBayesCore {
                                                     const std::string &cdt,
                                                     const std::vector<Taxon> &tn,
                                                     bool uo,
-                                                    Tree *t);  //!< Constructor
+                                                    Tree *t,
+                                                    const TypedDagNode<long> *age_check_precision);  //!< Constructor
```

Comment:
Why is this a DAG node? DAG nodes should only be used if the intention is that the variable might change during an analysis.